src/Foundation.Find.Cms/Models/Pages/PersonItemPage.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.ComponentModel.DataAnnotations;
+using EPiServer.Core;
+using EPiServer.DataAbstraction;
+using EPiServer.DataAnnotations;
+using EPiServer.Web;
+using Foundation.Cms.Pages;
+
+namespace Foundation.Find.Cms.Models.Pages
+{
+    [ContentType(DisplayName = "Person Item Page", 
+        GUID = "b5af511b-96c9-4ad7-828f-254924542430", 
+        Description = "Person Item Page shows info of specific person",
+        GroupName = FindTabNames.Person)]
+    [AvailableContentTypes(Availability.Specific, Exclude = new[] { typeof(PageData) })]
+    [ImageUrl("~/assets/icons/cms/blocks/contact.png")]
+    public class PersonItemPage : FoundationPageData
+    {
+        [CultureSpecific]
+        [Display(Name = "Job Title", GroupName = SystemTabNames.Content, Order = 1)]
```

Comment:
Please follow standard: Capitalisation (first letter in uppercase and the remaining letters in lowercase)
Update "Order" to 10-20-.....